### PR TITLE
Updated src/main, because on changing the FastAPI version on_event(st…

### PR DESCRIPTION
**Pull request by nargis-sultani**
…artup) was depecated in the new version.

Incorporated ragtech_commons_api and deleted the old ones

### CODE REVIEW

File: tests/api/routers/test_admin_api.py [hunk 1]
```diff
@@ -5,7 +5,10 @@
 from pytest_mock import MockerFixture
 from starlette.authentication import AuthCredentials
 
-from entities.models import AuthenticatedUser
+from regtech_api_commons.models.auth import AuthenticatedUser
+from regtech_api_commons.oauth2 import oauth2_admin
+
+oauth2_admin.OAuth2Admin.update_user
```
Comment (jcadam14) [inline tests/api/routers/test_admin_api.py:12]: Is this line supposed to be here?

Comment (nargis-sultani) [reply tests/api/routers/test_admin_api.py:12]: Ooops, nope. Removing it now.

**Review by jcadam14 — DISMISSED** [review]
Looks good now!

File: src/routers/admin.py [hunk 2]
```diff
@@ -21,12 +22,12 @@ def get_me(request: Request):
 @router.put("/me/", status_code=HTTPStatus.ACCEPTED, dependencies=[Depends(check_domain)])
 @requires("manage-account")
 def update_me(request: Request, user: UserProfile):
-    oauth2_admin.update_user(request.user.id, user.to_keycloak_user())
+    OAuth2Admin.update_user(request.user.id, user.to_keycloak_user())
```
Comment (lchen-2101) [inline src/routers/admin.py:25]: you should be creating an instance of the class, not use the class methods in a static fashion, not quite sure how this is working with the `_admin` attribute not constructed within the instance...

File: tests/app/test_config.py [hunk 3]
```diff
@@ -20,7 +21,7 @@ def test_jwt_opts_valid_values():
         "jwt_opts_test2": "true",
         "jwt_opts_test3": "12",
     }
-    settings = Settings(**mock_config)
+    settings = KeycloakSettings(**mock_config)
```
[the remark this answers is not in the document]
Comment (nargis-sultani) [reply tests/app/test_config.py:24]: OK, got it.

File: src/routers/institutions.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ async def create_institution(
     fi: FinancialInstitutionDto,
 ):
     db_fi = await repo.upsert_institution(request.state.db_session, fi)
-    kc_id = oauth2_admin.upsert_group(fi.lei, fi.name)
+    kc_id = OAuth2Admin.upsert_group(fi.lei, fi.name)
```
Comment (lchen-2101) [inline src/routers/institutions.py:52]: missed this one

File: tests/app/test_api_commons.py
Comment (lchen-2101) [inline tests/app/test_api_commons.py:1]: I meant removing the tests related to `KeycloakSettings` from `regtech-user-fi-management` repo completely, and add these tests to the `regtech-api-commons` repo at a later time (create a ticket for it, I think there might be a ticket already about adding tests and coverage to that repo)

**Review by lchen-2101 — APPROVED** [review]
LGTM

**Pull request merged**